lib/auth/auth_with_roles.go

```diff
@@ -22,7 +22,7 @@ import (
 	"time"
 
 	"github.com/gravitational/teleport"
-	"github.com/gravitational/teleport/lib/auth/proto"
+	proto "github.com/gravitational/teleport/api/auth"
```

Comment:
I would rename this package to `proto` instead of `auth`. You already have `auth` in `/lib` and this is always going to need to be renamed.